pkg/services/auth/jwt/auth.go

```diff
@@ -118,3 +120,17 @@ func HasSubClaim(jwtToken string) bool {
 
 	return claims.Subject != ""
 }
+
+func FindSubClaims(subclaim string, jsonMap any) (string, error) {
```

Comment:
I think it's better to use the same function that is used by the other providers: https://github.com/grafana/grafana/blob/c6a489cfd891387a65210a903e8cd652ac56974d/pkg/util/json.go#L59.

Reply:
I removed this function and replaced its uses with `SearchJSONForStringAttr`.